src/ElementsIsotopes.f90

```diff
@@ -105,6 +105,9 @@ MODULE ElementsIsotopes
       !> @copybrief ElementsIsotopes::getZAID_ElemIso
       !> @copydetails ElementsIsotopes::getZAID_ElemIso
       PROCEDURE,PASS :: getZAID => getZAID_ElemIso
```

Comment:
is getZAID still used?

Reply:
For now, yes.  Most of them are actually calculating the `ZZZAAAI` from the `ZAID` so they could get replaced, but it's quite a few locations so I intend to leave it for now.  And having this function doesn't hurt anyways.

Reply:
yeah